ROADMAP.md

```diff
@@ -6,62 +6,50 @@ As a public community project the KitOps roadmap is always open to feedback. If
 * Tell us in the [KitOps Discord](https://discord.gg/Tapeh8agYy) channel
 * Send us a message on [X / Twitter](https://twitter.com/Kit_Ops)
 
-## TL;DR
+## Roadmap
+🛳️ = Shipped
 
-The main things we're working on are:
+🏃‍➡️= In-progress
 
-* CLI dev mode to simplify running LLMs locally for testing or app integrations
-* Tutorials for using KitOps with LLM fine-tuning and RAG pipelines
-* Attestation for ModelKits and their contents
-* Signing of ModelKits
-* Model chaining (foundational --> tuned)
-* Deployment
-* More CLI distribution options
-* Documentation of the KitOps workflow
+### ModelKit Chaining 🛳️
 
-## Roadmap Details
+A ModelKit can refer to another ModelKit's model as its "base model." This is useful for situations where a user wants to take an existing model (e.g., Llama 3) but create their own model based on it. The new ModelKit would include a reference to the Llama 3 ModelKit and Kit would understand to pull both when the user asks for the new ModelKit.
 
-### Dev Mode (in-progress)
+* Add chaining to ModelKit schema 
+* Add support for chaining to relevant Kit CLI commands 
```

Comment:
There's trailing whitespace on these lines (causing the CI failure).

If you're using VS Code, you can trim trailing whitespace via pressing (cmd+k, cmd+x), or cmd+shift+p and searching for "trim trailing whitespace"